### PR TITLE
Incremental improvements to airline demo docs

### PR DESCRIPTION
**Pull request by mgasner**

### CODE REVIEW

File: python_modules/airline-demo/airline_demo/solids.py [hunk 1]
```diff
@@ -166,6 +166,7 @@ def thunk_database_engine(context):
 
 @solid(
     name='download_from_s3',
+    # inputs=[InputDefinition('files_to_download', dagster_type=List(Dict())],
```
Comment (schrockn) [inline python_modules/airline-demo/airline_demo/solids.py]: ?

File: python_modules/airline-demo/environments/local_full_download.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+solids:
+  download_archives:
```
Comment (schrockn) [inline python_modules/airline-demo/environments/local_full_download.yml]: We can do this in a separate diff, but should we model these an inputs and make skip_if_present a solid-wide config?

**Comment by mgasner** [comment]
Oops

File: python_modules/airline-demo/airline_demo/solids.py (hunk 1 above)
Comment (mgasner) [reply python_modules/airline-demo/airline_demo/solids.py]: Oops

File: python_modules/airline-demo/environments/local_full_download.yml (hunk 2 above)
Comment (mgasner) [reply python_modules/airline-demo/environments/local_full_download.yml]: Yeah, probably -- I'd like to document the current state of the repo and then make some improvements to the codebase.

**Comment by schrockn** [comment]
sgtm

Sent via Superhuman ( https://sprh.mn/?vip=schrockn@elementl.com )

**Pull request merged**